app/styles/_overview.less

```diff
@@ -97,59 +106,94 @@
   .standalone-service {
     width: 100%;
     @media (min-width: @screen-sm-min) {
-      width: 50%;
-
-      &:nth-child(odd) {
-          padding-right: 5px;
-      }
-
-      &:nth-child(even) {
-          padding-left: 5px;
-      }
-
+      .flex(@columns: 0 0 49%);
+    } 
+    @media (min-width: @screen-md-min) {
+      .flex(@columns: 0 0 49.375%);
```

Comment:
So something like flex-basis: 50% and margin-right: 15px doesn't work because the second tile wraps?